packages/vm/test/api/runTx.spec.ts

```diff
@@ -317,6 +319,24 @@ describe('runTx() -> API parameter usage/data errors', () => {
     assert.deepEqual(res.accessList, [])
   })
 
+  it('simple run (reportPreimages option)', async () => {
+    const vm = await VM.create({ common })
+
+    const tx = getTransaction(vm.common, 0, true)
+
+    const caller = tx.getSenderAddress()
+    const acc = createAccount()
+    await vm.stateManager.putAccount(caller, acc)
+
+    const res = await vm.runTx({ tx, reportPreimages: true })
+
+    const hashedCallerKey = vm.stateManager.getAppliedKey(caller.bytes)
+
+    const retrievedPreimage = res.preimages?.get(bytesToHex(hashedCallerKey))
+
+    assert.ok(retrievedPreimage !== undefined && equalsBytes(retrievedPreimage, caller.bytes))
```

Comment:
This tests preimages for accounts, I would also add a test here where storage slot preimages are saved. Do we also need it for the code hashes? (I don't think so - right? In context of the Verkle transition)

Reply:
We have added some additional test vectors from the client-side. Notably we also needed to test blockWitnesses and coinbase, and I think the txs that are being tested at the client level would also indirectly test storage slot preimages. I feel like it's fine to leave that runTx test relatively minimal (basically just testing that the reportPreiamges option results in preimages being added to the journal and then to the txResult), wdyt?

Reply:
yes i believe the client test has a tx accessing storage